oauth2client/contrib/gce.py

```diff
@@ -51,13 +59,17 @@ class AppAssertionCredentials(AssertionCredentials):
     """
 
     @util.positional(2)
-    def __init__(self, scope, **kwargs):
+    def __init__(self, scope='', **kwargs):
         """Constructor for AppAssertionCredentials
 
         Args:
             scope: string or iterable of strings, scope(s) of the credentials
```

Comment:
We're not in a hole. GCE never honored the scopes and now we're just adding a warning for it (and also not sending it in the 

```
http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token
```

request. The only other change is to make it so that users don't **need** to pass scopes (since they don't do anything). But the rest of the behavior (i.e. for old code that does pass scopes) will work just fine.